<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Face recognizer</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-recognizer">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>Operate(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menuRegister">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Register</string>
     </property>
     <property name="icon">
      <iconset theme="register">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionRegisterImage"/>
     <addaction name="actionRegisterImage_directory"/>
     <addaction name="actionRegisterVideo"/>
    </widget>
    <widget class="QMenu" name="menuRecognizer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Recognizer</string>
     </property>
     <property name="icon">
      <iconset theme="face-recognizer">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <addaction name="actionRecognizerImage"/>
     <addaction name="actionRecognizerVideo"/>
    </widget>
    <widget class="QMenu" name="menuAI_libraries">
     <property name="title">
      <string>AI libraries</string>
     </property>
     <property name="icon">
      <iconset theme="tools">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
    <addaction name="actionStart"/>
    <addaction name="separator"/>
    <addaction name="actionDisplay"/>
    <addaction name="menuRegister"/>
    <addaction name="menuRecognizer"/>
    <addaction name="separator"/>
    <addaction name="menuAI_libraries"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menuCamer_Orientation">
     <property name="title">
      <string>Camer Orientation</string>
     </property>
     <addaction name="action0"/>
     <addaction name="action90"/>
     <addaction name="action180"/>
     <addaction name="action270"/>
    </widget>
    <addaction name="actionSet_model_path"/>
    <addaction name="actionOptions_O"/>
    <addaction name="menuCamer_Orientation"/>
    <addaction name="actionKeep_display_aspect_ratio"/>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
    <addaction name="actionUpdate_U"/>
   </widget>
   <widget class="QMenu" name="menuSource">
    <property name="title">
     <string>Source(&amp;S)</string>
    </property>
    <addaction name="actionCamera"/>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menuOperate"/>
   <addaction name="menuSource"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
  </widget>
  <action name="actionStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="statusTip">
    <string>Start</string>
   </property>
  </action>
  <action name="actionDetector">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Detector</string>
   </property>
   <property name="statusTip">
    <string>Detector</string>
   </property>
  </action>
  <action name="actionLandmarker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Landmarker</string>
   </property>
   <property name="statusTip">
    <string>Landmarker</string>
   </property>
  </action>
  <action name="actionSet_model_path">
   <property name="icon">
    <iconset theme="folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Set model path</string>
   </property>
   <property name="statusTip">
    <string>Set model path</string>
   </property>
  </action>
  <action name="actionAbout_A">
   <property name="icon">
    <iconset theme="face-recognizer">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
   <property name="iconText">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update</string>
   </property>
  </action>
  <action name="actionDisplay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="video-display">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
   <property name="iconText">
    <string>Display</string>
   </property>
   <property name="toolTip">
    <string>Display</string>
   </property>
   <property name="statusTip">
    <string>Display</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="camera-video">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="video-file">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionRegisterImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="image">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="toolTip">
    <string>Register from image</string>
   </property>
   <property name="statusTip">
    <string>Register from image</string>
   </property>
   <property name="whatsThis">
    <string>Register from image</string>
   </property>
  </action>
  <action name="actionRegisterImage_directory">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="folder">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Image directory</string>
   </property>
   <property name="toolTip">
    <string>Register from image directory</string>
   </property>
   <property name="statusTip">
    <string>Register from image directory</string>
   </property>
   <property name="whatsThis">
    <string>Register from image directory</string>
   </property>
  </action>
  <action name="actionRegisterVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="video-display">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>Register from video</string>
   </property>
   <property name="statusTip">
    <string>Register from video</string>
   </property>
   <property name="whatsThis">
    <string>Register from video</string>
   </property>
  </action>
  <action name="actionRecognizerImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="image">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="toolTip">
    <string>Recognizer from image</string>
   </property>
   <property name="statusTip">
    <string>Recognizer from image</string>
   </property>
   <property name="whatsThis">
    <string>Recognizer from image</string>
   </property>
  </action>
  <action name="actionRecognizerVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="video-display">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
   <property name="toolTip">
    <string>Recongnizer from video</string>
   </property>
   <property name="statusTip">
    <string>Recongnizer from video</string>
   </property>
   <property name="whatsThis">
    <string>Recongnizer from video</string>
   </property>
  </action>
  <action name="actionOptions_O">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="applications-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Options(&amp;O)</string>
   </property>
  </action>
  <action name="action90">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>90°</string>
   </property>
  </action>
  <action name="action180">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>180°</string>
   </property>
  </action>
  <action name="action270">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>270°</string>
   </property>
  </action>
  <action name="action0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>0°</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="whatsThis">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionKeep_display_aspect_ratio">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep display aspect ratio</string>
   </property>
   <property name="statusTip">
    <string>Keep display aspect ratio</string>
   </property>
   <property name="whatsThis">
    <string>Keep display aspect ratio</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
